bower.json

```diff
@@ -1,16 +1,31 @@
 {
-    "name": "angular-recaptcha",
-    "version": "2.1.1",
-    "keywords": ["angular", "captcha", "recaptcha", "vividcortex", "human", "form", "validation", "signup", "security", "login"],
-    "main": "release/angular-recaptcha.js",
-    "ignore": [
-        "**/.*",
-        "Gruntfile.js",
-        "src",
-        "node_modules",
-        "bower_components"
-    ],
-    "dependencies": {
-        "angular": "1.*"
-    }
+  "name": "angular-recaptcha",
+  "version": "2.1.1",
+  "keywords": [
+    "angular",
+    "captcha",
+    "recaptcha",
+    "vividcortex",
+    "human",
+    "form",
+    "validation",
+    "signup",
+    "security",
+    "login"
+  ],
+  "main": "release/angular-recaptcha.js",
+  "ignore": [
+    "**/.*",
+    "Gruntfile.js",
+    "src",
+    "node_modules",
+    "bower_components"
+  ],
+  "dependencies": {
+    "angular": "1.*"
+  },
+  "devDependencies": {
+    "angular-mocks": "~1.*",
+    "jquery": "~2.1.3"
```

Comment:
Yeah, for some reason, I was getting the jqlite noSel error. Since when I used this module, I also had jQuery as a dependency, it'd not throw any errors.

Here are the lines, by the way (50 and 77, in directive.js):

```js
angular.element('.pls-container').parent().remove();
```